pkg/fixtures/purger_s3.go

```diff
@@ -0,0 +1,44 @@
+package fixtures
+
+import (
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/cloud/aws/s3"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/aws/aws-sdk-go/aws"
+	aws_s3 "github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+type s3Purger struct {
+	logger     mon.Logger
+	client     s3.Client
+	bucketName *string
+}
+
+func newS3Purger(config cfg.Config, logger mon.Logger, settings *s3.Settings) *s3Purger {
+	client := s3.ProvideClient(config, logger, settings)
+
+	return &s3Purger{
+		logger: logger,
+		bucketName:   &settings.BucketName,
+		client: client,
+	}
+}
+
+func (p *s3Purger) purge() error {
+	p.logger.Infof("purging bucket %s", *p.bucketName)
+	iter := s3manager.NewDeleteListIterator(p.client, &aws_s3.ListObjectsInput{
```

Comment:
Nope a bucket needs to be emptied first. Otherwise a `ErrCodeBucketNotEmpty` will arise, and there is no other way to force delete a non empty bucket.

Reply:
Ahh, it seems the --force option on the cli is not an input option for the request, didn't see that.